src/plugins/discover/public/application/main/components/layout/use_discover_histogram.ts

```diff
@@ -399,8 +391,8 @@ export const useDiscoverHistogram = ({
     dataView: isEsqlMode ? esqlDataView : dataView,
     query: isEsqlMode ? esqlQuery : query,
     filters: filtersMemoized,
-    timeRange: timeRangeMemoized,
-    relativeTimeRange,
+    timeRange,
+    relativeTimeRange: timeRangeRelative,
```

Comment:
Can we have just one time range which is in absolute format? The relative one was probably necessary in the histogram for the previous approach but now it might be redundant.

Reply:
I had a look at it, but I don't think we can currently do this because we are using the relative time range for e.g. the edit visualization 

https://github.com/elastic/kibana/blob/61d0320c6422116dcf1c4e26f8f80760d7a3bb81/src/plugins/unified_histogram/public/layout/layout.tsx#L84-L87

I think that's used e.g. for the time range when clicking on the Lens button in DataView mode